scripts/minici-presubmit-all-tests.sh

```diff
@@ -22,7 +22,7 @@ case $1 in
         export TIMEOUT="2h15m"
         ;;
     minishift)
-        export JOB_NAME="odo-minishift-pr-tests"
+        export JOB_NAME="odo-minishift-pr-build"
         export SENDQUEUE="amqp.ci.queue.minishift.send"
```

Comment:
@prietyc123 this change is going in with #4561 Lets avoid duplication of effort, I am focusing on minishift